frontend/public/components/pod-exec.jsx

```diff
@@ -141,6 +142,10 @@ export class PodExec extends React.PureComponent {
     this.ws && this.ws.send(`4${data}`);
```

Comment:
not your change, but curious what the leading `4` does here.

Reply:
That really needs a comment, doesn't it?